Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -108,19 +109,33 @@
                     listener="#{DataEditorForm.structurePanel.onNodeCollapse}"/>
             <p:ajax event="expand"
                     listener="#{DataEditorForm.structurePanel.onNodeExpand}"/>
-            <p:treeNode expandedIcon="#{logicalNode.dataObject['class'].simpleName eq 'View' ? 'ui-icon-document' : 'ui-icon-folder-expanded'}"
-                        collapsedIcon="#{logicalNode.dataObject['class'].simpleName eq 'View' ? 'ui-icon-document' : 'ui-icon-folder-collapsed'}"
+            <p:treeNode type="Structure" expandedIcon="ui-icon-folder-expanded"
+                        collapsedIcon="ui-icon-folder-collapsed"
                         styleClass="#{logicalNode.linked ? 'linked' : ''} #{logicalNode.undefined ? 'undefined' : ''}">
                 <!--@elvariable id="logicalElementType" type="java.lang.String"-->
                 <ui:param name="logicalElementType" value="#{logicalNode.label}"/>
                 <!--@elvariable id="defaultTreeNode" type="org.primefaces.model.DefaultTreeNode"-->
-                <h:outputText rendered="#{DataEditorForm.structurePanel.showHierarchyLevel and logicalNode.dataObject['class'].simpleName ne 'View'}"
+                <h:outputText rendered="#{DataEditorForm.structurePanel.showHierarchyLevel}"
                               value="&lt;#{fn:length(fn:split(defaultTreeNode.rowKey, '_'))}&gt; "/>
                 <h:outputText value="#{empty logicalElementType ? msgs['dataEditor.withoutType'] : logicalElementType}"
                               title="#{logicalNode.undefined ? msgs['dataEditor.undefinedStructure'] : ''}"
-                              rendered="#{DataEditorForm.structurePanel.titleMetadata eq 'type' or logicalNode.dataObject['class'].simpleName eq 'View'}"/>
+                              rendered="#{DataEditorForm.structurePanel.titleMetadata eq 'type'}"/>
                 <h:outputText value="#{DataEditorForm.getStructureElementTitle(logicalNode.dataObject)}"
-                              rendered="#{DataEditorForm.structurePanel.titleMetadata ne 'type' and logicalNode.dataObject['class'].simpleName ne 'View'}"/>
+                              rendered="#{DataEditorForm.structurePanel.titleMetadata ne 'type'}"/>
+                <h:outputText value="#{DataEditorForm.structurePanel.getMultipleAssignmentsIndex(logicalNode) + 1}"
+                              rendered="#{logicalNode.assignedSeveralTimes}"
+                              styleClass="assigned-several-times"/>
```

Comment:
```suggestion
```
Only `View`s can be "assigned multiple times", not structure elements, if I am not mistaken.